pkgs/applications/networking/browsers/qutebrowser/default.nix

```diff
@@ -1,25 +1,27 @@
-{ stdenv, fetchurl, python, buildPythonPackage, qt5, pyqt5, jinja2, pygments, pyyaml, pypeg2}:
+{ stdenv, fetchgit, python, buildPythonPackage, qt5, pyqt5, jinja2, pygments, pyyaml, pypeg2 }:
 
-let version = "0.2.1"; in
+let version = "0.3-pre"; in
```

Comment:
Well, the point of `0.3-pre` is that I believe that’s _actually_ the commit that will become 0.3. But, fine, I’ll change to the date.